README.md

```diff
@@ -34,10 +34,13 @@ generated file by hand.
 
 ## Usage
 
+In a Unix or Node.js command prompt, use:
```

Comment:
Please can you replace this with "Create a WebKit block list `output.json` from the Adblock Plus filter list `input.txt`:"?